hphp/test/slow/include/fast-stat-cache.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+$file = tempnam('/tmp', 'test-fast-stat-cache');
+file_put_contents($file, "<?php\necho 'Old';");
+include($file);
+unlink($file);
+sleep(1);
```

Comment:
I understand the intention of sleep() here.  However, it is racy because the operating system may decide to wake up the process later than the specified time.  In the unlikely case that the OS scheduling delay is more than a second, the result could be different from expected here (and this may happen without the first sleep()). 

If we clear the cache any time we make changes to the file system (create/delete a file, as opposed to checking existence), would things work better in this aspect?